### DISCUSSION TITLE
Error uploading file for knowledge

### DISCUSSION THREAD

**Discussion by mazierovictor** [open-webui/open-webui#6214] [General]
# Bug Report

I am trying to attach files in Workspace > Knowledge and I am getting error 400: 'NoneType' object has no attribute 'ids' and I am not able to resolve it

## Installation Method

My installation is docker based

## Environment

- **Open WebUI Version:** v0.3.32
- I am using Groq as AI model

- **Operating System:** Ubuntu Linux 24.04.1 (Server)

**Confirmation:**

- [x ] I have read and followed all instructions provided in README.md.
- [x ] I am using the latest version of Open WebUI and Ollama.
- [ x ] I have included the browser console logs.
- [ x ] I have included the Docker container logs.
- [x ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected behavior:

I expected the file to be attached to the knowledge base without any major problems.

## Actual behavior:

When trying to upload the file to the knowledge base, whether it is pdf and/or txt, the error displayed is 400: 'NoneType' object has no attribute 'ids' during the upload.

## Description

**Bug summary:**
Given the above, the error occurs when trying to attach files within the Workspace > Knowledge Menu, when creating a knowledge base and trying to upload any file, the error 400: 'NoneType' object has no attribute 'ids' occurs during the upload and the system crashes, causing the page to need to be refreshed.

## Reproduction details

**Steps to reproduce:**
[Describe the steps to reproduce the bug. Please be as detailed as possible.]

## Logs and Screenshots

**Browser console logs:**
[logs_browse.txt](https://github.com/user-attachments/files/17388020/logs_browse.txt)

**Docker container logs:**
[logs_docker.txt](https://github.com/user-attachments/files/17388021/logs_docker.txt)

**Screenshots/screen recordings (if applicable):**
![image](https://github.com/user-attachments/assets/a7c81db5-9ca7-40fe-9d94-cb94268af960)

**Comment by mazierovictor** [comment]
The error was caused because I was using a hugging face Embedding, when I returned to the standard Embedding model engine the error no longer existed.

So there is some problem with using third party Embedding.

**Reply by McNickSistoPro** [reply]
Did you figure this out ?

**Reply by pdob-git** [reply]
Which embedding model have you used to make it work?

**Comment by lundberg-johan12** [comment]
This happened to me when using an outdated OPENAI API KEY
